README.md

```diff
@@ -24,6 +24,14 @@ pip install --upgrade beets>=1.3.13
 pip install git+git://github.com/geigerzaehler/beets-alternatives.git@master
 ```
 
+Now you have to enable this plugin in your beets configuration file (normally ~/beets/config.yaml on linux systems):
```

Comment:
Instead of referring to the actual file path, we should tell the user to run [`beet config -e`](http://beets.readthedocs.org/en/v1.3.13/reference/cli.html#config).